### ISSUE TITLE
Multiple editors cause overlap of toolbar.

### ISSUE THREAD

**Issue by eujc21** [globocom/megadraft#91]
**Issue:** When rendering multiple editors in a component, the tool bar overlaps when editing.

**Replicate Issue:** Render two MegaDrafEditor components in one layout:
```
import React from 'react';
import { MegadraftEditor } from 'megadraft';

const Header = (props) => (
  <MegaDraftEditor />
);
const Body = (props) => (
  <MegaDraftEditor />
)
export default class Testing extends React.Component {
  constructor(props){
    super(props)
  }
  render() {
    <div>
      <Header />
      <Body />
    </div>
  }
}
```
![pasted image at 2016_11_01 09_12 pm](https://cloud.githubusercontent.com/assets/973157/19916993/848393aa-a095-11e6-91c7-6d51bcf1d6a5.png)

**Comment by ryuheiyokokawa** [comment]
Bump.  I'm having the same issue.

**Comment by ricobl** [comment]
Hey @eujc21 and @ryuheiyokokawa!

I've created the [multiple-editors-test branch](https://github.com/globocom/megadraft/tree/multiple-editors-test) with a second editor on the demo page and wrote a fix for both the sidebar and the toolbar.

Looks like it's working but I didn't have the time to write tests.

If anyone wants to help, the branch is a starting point.

**Comment by ryuheiyokokawa** [comment]
@ricobl Thanks for that!  I'll try it out later today.

**Comment by ryuheiyokokawa** [comment]
Took me forever (like a few months) to get to this, but i did test the code.  I pulled master (0.4.26) into the fork of this branch and things seem to be working well.  You can checkout my janky setup with my fork (https://github.com/ryuheiyokokawa/megadraft/tree/multiple-editors-test), but that's purely for me to be able to pull from NPM (hence took out a few gitignores, etc).  Thank you again for the work @ricobl !  Its just difficult when you don't have working knowledge of a code base.

PS: This code base required Node 8 / NPM 5.  Not sure if that's explicitly stated in package.json or the README, but it wasn't an issue with the use of NVM.

**Comment by ricobl** [comment]
Just as an update, #163 fixes the formatting toolbar overlap by hiding it on blur.

The sidebar is still a problem though...

**Comment by ricobl** [comment]
That was quick! Thanks to @dchristensen the fix for the sidebar is released under the `0.4.31` version.
